### ISSUE TITLE
Could you  please share me the Eval results?

### ISSUE THREAD

**Issue by ngoanpv** [brightmart/albert_zh#83]
Hi @brightmart ,  could you share my your eval results.

 I am pre-training on my own language and confused after 900000 steps, my eval results is:

***** Eval results *****
global_step = 900000
loss = 1.7051821
masked_lm_accuracy = 0.65745
masked_lm_loss = 1.6958559
sentence_order_accuracy = 0.9998438
sentence_order_loss = 0.001201547

The masked_lm_loss is not good.

Thanks.

**Comment by brightmart** [comment]
hi. i think the result is fine. the main reason is that there are many possible true candidate for a masked word in real word. such as: 
i like eating apple
i like eating vegetables
although in one real sentence the true answer is apple, but other words also feasible.
if i am not wrong, the best possible masked lm accuracy on eval or dev set could be around 0.7 for best albert models. 

so your result is fine. also you still have methods to improve the result.

**Comment by ngoanpv** [comment]
Thank you @brightmart  for your explanation
To improve, i think I have to train more step / add more data. 
Do you have any suggestion?
